quickjs.h

```diff
@@ -705,6 +705,7 @@ JS_EXTERN int JS_SetPrototype(JSContext *ctx, JSValue obj, JSValue proto_val);
 JS_EXTERN JSValue JS_GetPrototype(JSContext *ctx, JSValue val);
 JS_EXTERN int JS_GetLength(JSContext *ctx, JSValue obj, int64_t *pres);
 JS_EXTERN int JS_SetLength(JSContext *ctx, JSValue obj, int64_t len);
+JS_EXTERN JSValue JS_ObjectSeal(JSContext *ctx, JSValue obj, int freeze);
```

Comment:
Is freeze a Boolean?

Reply:
yes, should I use JS_BOOL?

Reply:
done.
in the code it's called freeze_flag, but as far as I can tell it is used as bool